autoload/lsp/omni.vim

```diff
@@ -182,14 +184,27 @@ function! lsp#omni#get_vim_completion_item(item, ...) abort
         let l:word = s:remove_typed_part(l:word)
     endif
     let l:kind = lsp#omni#get_kind_text(a:item)
+
```

Comment:
Also might be worth totally removing inserText since it is deprecated. 
https://github.com/prabirshrestha/vim-lsp/issues/328#issuecomment-473724798

Reply:
Should I remove insertText on this Pull Request?

Reply:
Let us remove insertText. t has caused more problems to us than it was meant to solve.

Reply:
//cc @yami-beta.

Reply:
//cc @mlen since it was originally added in https://github.com/prabirshrestha/vim-lsp/pull/45

Reply:
> Let us remove insertText. t has caused more problems to us than it was meant to solve.

I agree with.